brainiak/isfc.py

```diff
@@ -68,6 +68,12 @@ def isc(D, collapse_subj=True, return_p=False, num_perm=1000,
         A random number generator instance to define the state of the
         random permutations generator.
 
+    float_type : either float16, float32, or float64,
+        depending on the required precision,
+        and available memory in the system.
+        cast all the arrays generated during the execution to
+        specified float type, in order to save memor

```

Comment:
Typo: "memor". Also, please write regular sentences in the documentation, i.e., starting with a capital, full stop only at the end.

Reply:
Thank you, fixed accordingly.